include/experimental/__p0009_bits/compressed_pair.hpp

```diff
@@ -28,8 +28,8 @@ namespace detail {
 // For no unique address emulation, this is the case taken when neither are empty.
 // For real `[[no_unique_address]]`, this case is always taken.
 template <class _T1, class _T2, class _Enable = void> struct __compressed_pair {
-  _MDSPAN_NO_UNIQUE_ADDRESS _T1 __t1_val;
-  _MDSPAN_NO_UNIQUE_ADDRESS _T2 __t2_val;
+  _MDSPAN_NO_UNIQUE_ADDRESS _T1 __t1_val{};
+  _MDSPAN_NO_UNIQUE_ADDRESS _T2 __t2_val{};
   MDSPAN_FORCE_INLINE_FUNCTION _MDSPAN_CONSTEXPR_14 _T1 &__first() noexcept { return __t1_val; }
```

Comment:
It looks like we still need `_MDSPAN_CONSTEXPR_14`, as `_MDSPAN_USE_CONSTEXPR_14` suggests the need for a work-around with Intel compilers.  Is that right?

https://github.com/search?q=repo%3Akokkos%2Fmdspan+_MDSPAN_USE_CONSTEXPR_14&type=code

Reply:
no idea I did NOT attempt to refactor anything here.